src/capitalsFirst.js

```diff
@@ -21,7 +21,30 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const words = str.split(' ');
+  const upperCaseWords = [];
+  const lowerCaseWords = [];
+  const FIRST_LOWERCASE_LETTER = 'a';
+  const LAST_LOWERCASE_LETTER = 'z';
+  const filteredWords = words.filter((word) => {
+    return isLowercaseLetter(word) <= LAST_LOWERCASE_LETTER &&
+        isLowercaseLetter(word) >= FIRST_LOWERCASE_LETTER;
+  });
+  filteredWords.forEach((word) => {
+    if (isUpperCase(word[0])) {
+      upperCaseWords.push(word);
+    } else {
+      lowerCaseWords.push(word);
+    }
+  });
+  return [...upperCaseWords, ...lowerCaseWords].join(' ');
+}
+
+function isUpperCase(letter) {
+  return letter.toUpperCase() === letter;
+}
+function isLowercaseLetter(wd) {
```

Comment:
`wd`? It is like `ch` - I do not know what it means. Be consistent name variable letter as above